src/java.base/share/classes/java/lang/foreign/GroupLayout.java

```diff
@@ -82,73 +82,81 @@ public sealed interface GroupLayout extends MemoryLayout permits StructLayout, U
      * of the provided {@link Record} {@code type} by means of matching the names of the
      * record components with the names of the elements in this group layout}
      * <p>
-     * The mapping between the record type and this memory layout is defined as follows;
+     * A mapper {@code M} between a record type {@code R}, a memory layout {@code L} and an {@code offset}
+     * (which is zero for the initial invocation of this method) is defined as follows {@code M(type, layout, offset)};
      * <p>
-     * Let <em>R</em> be the provided record {@code type} with the constituent components
-     * <em>C<sub>0</sub></em>, ..., <em>C<sub>N-1</sub></em>, where <em>N</em> is non-negative.
+     * Let {@code R} comprise its the constituent components
```

Comment:
Maybe simplify: "Let C1, C2 ... Cn" be the components of R (with N non-negative)"